src/app/Overview/Overview.tsx

```diff
@@ -33,10 +35,10 @@ const AggregateStatusCards: React.FunctionComponent = () => {
       terminated: inventoryData?.clusters?.archived || 0,
     },
     instancesByStatus: {
-      running: inventoryData?.instances?.running || 0,
-      stopped: inventoryData?.instances?.stopped || 0,
-      unknown: inventoryData?.instances?.unknown || 0,
-      terminated: inventoryData?.instances?.archived || 0,
+      running: 0, // Not available in current API
+      stopped: 0, // Not available in current API
+      unknown: 0, // Not available in current API
+      terminated: 0, // Not available in current API
     },
```

Comment:
Nice idea to comment this by the moment.  Remember to remove them when the integration is Ready